infrastructure/modules/aks/providers.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "< 4.39.0"
+      version = "< 4.50.0"
```

```suggestion
      version = "< 4.39.0"
```
Issue is still not fixed keeping <4.39.0